_format/1.1/index.md

```diff
@@ -1778,8 +1816,456 @@ If a server encounters a query parameter that does not follow the naming
+  It is used to support old servers that don't understand the profile parameter.
+
+Servers **MAY** add profiles to a JSON API document even if the client has not
+requested them. The recipient of a document **MUST** ignore any profile extensions
+in that document that it does not understand. The only exception to this is profiles
+whose support is required using the `profile` query parameter, as described later.
+
+#### <a href="#profiles-sending" id="profiles-sending" class="headerlink"></a> Sending Profiled Documents
+
+Clients and servers **MUST** include the `profile` media type parameter in
+conjunction with the JSON API media type in a `Content-Type` header to indicate
+that they have applied one or more profiles to a JSON API document.
+
+Likewise, clients and servers applying profiles to a JSON API document **MUST**
+include a [top-level][top level] [`links` object][links] with a `profile` key,
+and that `profile` key **MUST** include a [link] to the URI of each profile
+that has been applied.
+
+When an older JSON API server that doesn't support the `profile` media type
+parameter receives a document with one or more profiles, it will respond with a
+`415 Unsupported Media Type` error.
+
+After attempting to rule out other possible causes of this error, a client that
+receives a `415 Unsupported Media Type` **SHOULD** remove the profile extensions
+it has applied to the document and retry its request without the `profile` media
+type parameter. If this resolves the error, the client **SHOULD NOT** attempt to
+use profile extensions in subsequent interactions with the same API.
+
+> The most likely other causes of a 415 error are that the server doesn't
+support JSON API at all or that the client has failed to provide a required
+profile.
+
+### <a href="#profile-query-parameter" id="profile-query-parameter" class="headerlink"></a> `profile` Query Parameter
+
+A client **MAY** use the `profile` query parameter to _require_ the server to
+apply one or more profiles when processing the request. The value of the `profile`
+query parameter **MUST** equal a URI-encoded whitespace-separated list of profile URIs.
+
+If a server receives a request requiring the application of a profile or
+combination of profiles that it can not apply, it **MUST** respond with a `400
+Bad Request` status code.
+
+> Note: When a client lists a profile in the `Accept` header, it's asking the
+> server to compute its response as normal, but then send the response document
+> with some extra information, as described in the requested profile. By
+> contrast, when a client lists a profile in the `profile` *query parameter*,
+> it's asking the server to *process the incoming request* according to the
+> rules of the profile. This can fundamentally change the meaning of the 
+> server's response.
+
+#### <a href="#profile-query-parameter-omitting" id="profile-query-parameter" class="headerlink"></a> Omitting the `profile` Query Parameter
+
```

Comment:
I wonder if this section would be more clear with a motivating lead-in, such as "A server may wish to apply a set of default profiles without requiring them to be explicitly requested."

Reply:
I like the idea of a motivating lead in makes sense to me, and I've added one in the latest commit. I used slightly different wording because I didn't want to imply that the server could alter the HTTP resource's behavior by applying a profile if _no_ query param is specified.